services/QuillLMS/spec/models/milestone_spec.rb

```diff
@@ -16,9 +16,16 @@
 require 'rails_helper'
 
 describe Milestone, type: :model do
+  before do
```

Comment:
Let's use `subject { build(:milestone) }` instead of the before block.  This will prevent an unnecessary creation of milestone records in the test.